app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -49,6 +53,8 @@ public static Intent newIntent(Context context, int resItem) {
 				}
 			});
 			bottomNavigation.setSelectedItemId(getIntent().getIntExtra(RES_ITEM, R.id.dashboard));
+			if (getPreferenceManager().isManualUploadDialog())
+				ConfirmDialogFragment.newInstance(null, getString(R.string.Modal_ManualUpload_Title), getString(R.string.Modal_ManualUpload_Paragraph), null, getString(R.string.Modal_ManualUpload_Enable), getString(R.string.Modal_ManualUpload_Disable), null).show(getSupportFragmentManager(), null);
```

Comment:
This line should be wrapped to a more reasonable column width to improve readability.

Reply:
The fingerprint of this function also seems pretty tricky, perhaps some improvements can be made to make the argument count smaller?

Having a functions which take many arguments is generally a pretty strong sign of code smell documented in countless programming best practices:
* https://refactoring.guru/smells/long-parameter-list
* https://www.matheus.ro/2018/01/29/clean-code-avoid-many-arguments-functions/
* https://stackoverflow.com/questions/174968/how-many-parameters-are-too-many

I think, especially in light of the fact there are several `null` arguments this could benefit from perhaps making use of setter methods?

Reply:
line wrapped. about "setter methods" i agree but i suggest to postpone in a second moment

Reply:
I opened a ticket for this: https://github.com/ooni/probe-android/issues/218